ospfd/ospf_te.c

```diff
@@ -2576,7 +2577,10 @@ DEFUN (show_ip_ospf_mpls_te_link,
 	}
 	/* Interface name is specified. */
 	else {
-		ifp = if_lookup_by_name_all_vrf(argv[idx_interface]->arg);
+		if (ospf)
+			ifp = if_lookup_by_name(
+					argv[idx_interface]->arg,
+					ospf->vrf_id);
 		if (ifp == NULL)
 			vty_out(vty, "No such interface name\n");
 		else
```

Comment:
The output should be enhance with the VRF information as a user performing a `sh ip ospf mpls_te interface XXX` will get a `No such interface name` if interface XXX is not belonging to default VRF. 

In fact, looking more carefully to the code, it is not possible to perform a `sh ip ospf vrf XX mpls_te interface YY` to obtain the MPLS TE information of interface YY within vrf XX.